csrc/mmdeploy/codebase/mmpose/cuda/topdown_affine.cpp

```diff
@@ -0,0 +1,69 @@
+// Copyright (c) OpenMMLab. All rights reserved.
+
+#include "mmdeploy/codebase/mmpose/topdown_affine.h"
+
+#include <vector>
+
+#include "ppl/cv/cuda/crop.h"
+#include "ppl/cv/cuda/resize.h"
+
+namespace mmdeploy::mmpose {
+
+class CropAndResizeCudaImpl : public CropAndResize {
```

Comment:
When fusing multiple operations, like Crop, Resize, and Pad. We don't want to name it "CropAndResizeAndPad". Chaining the names of the fused ops directly like "CropResizePad" should be more maintainable. 

Here `CropResize` would be a better name for the fused Crop and Resize operation.